### PR TITLE
Doc edits from QE check

### PR DESCRIPTION
**Pull request by PaulRMellor**
Signed-off-by: prmellor <pmellor@redhat.com>

Documentation

Couple of small edits following QE check of install and upgrade procedures.

1. Upgrading the Cluster Operator to a later version now includes link to download latest install files in prereqs.
2. Plus - I've moved a mirrormaker2 sub-section, to make reuse easier.
3. Edit for oauth to make a sentence clearer.

### Checklist

_Please go through this checklist and make sure all applicable tasks have been done_

- [ ] Update/write design documentation in `./design`
- [ ] Write tests
- [ ] Make sure all tests pass
- [x] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

### CODE REVIEW

File: documentation/shared/attributes.adoc [hunk 1]
```diff
@@ -21,7 +21,7 @@
 
 // Kafka upgrade attributes used in kafka upgrades section
 :DefaultKafkaVersion: 2.4.0
-:KafkaVersionLower: 2.3.1
+:KafkaVersionLower: 2.3.0
```
Comment (scholzj) [inline documentation/shared/attributes.adoc]: `2.3.1` was the correct version here.

File: documentation/modules/proc-upgrading-the-cluster-operator.adoc [hunk 2]
```diff
@@ -5,11 +5,12 @@
 [id='proc-upgrading-the-co-{context}']
 = Upgrading the Cluster Operator to a later version
 
-This procedure describes how to upgrade a Cluster Operator deployment to a later version.
+This procedure describes how to upgrade a Cluster Operator deployment to a later version
```
Comment (scholzj) [inline documentation/modules/proc-upgrading-the-cluster-operator.adoc]: Out of curiosity ... why should the `.` not be here?

File: documentation/assemblies/mirrormaker2/assembly-mirrormaker.adoc [hunk 3]
```diff
@@ -21,5 +21,8 @@ NOTE: MirrorMaker 2.0 has features not supported by the previous version of Mirr
 //Describes the underlying archiecture and how it is used in replication
 include::modules/con-mirrormaker-replication.adoc[leveloffset=+1]
 
+//Handling of ACLs in replication
+include::modules/con-mirrormaker-acls.adoc[leveloffset=+1]
```
Comment (scholzj) [inline documentation/assemblies/mirrormaker2/assembly-mirrormaker.adoc]: Thsi file seems to be missing.

Comment (PaulRMellor) [reply documentation/assemblies/mirrormaker2/assembly-mirrormaker.adoc]: It's added now!

File: documentation/modules/proc-upgrading-the-cluster-operator.adoc (hunk 2 above)
Comment (PaulRMellor) [reply documentation/modules/proc-upgrading-the-cluster-operator.adoc]: It should. And I've returned it.

File: documentation/shared/attributes.adoc (hunk 1 above)
Comment (PaulRMellor) [reply documentation/shared/attributes.adoc]: Yes. I assumed a change was required here, but I've reverted back now.

File: documentation/modules/proc-upgrading-the-cluster-operator.adoc [hunk 4]
```diff
@@ -9,7 +9,8 @@ This procedure describes how to upgrade a Cluster Operator deployment to a later
 
 .Prerequisites
 
-* An existing Cluster Operator deployment.
+* An existing Cluster Operator deployment is available.
+* The installation files have been xref:downloads-{context}[downloaded for the new version].
```
Comment (laidan6000) [inline documentation/modules/proc-upgrading-the-cluster-operator.adoc]: Consider the active voice:

* You have downloaded the [installation files](#) for the new version.

**Comment by PaulRMellor** [comment]
Please merge and cherry-pick

**Pull request merged**